.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -17,6 +19,8 @@ schedules:
     branches:
       include:
         - master
+        - release/*
+        - hotfix/*
```

Comment:
I don't know if we want to necessarily run these branches every night? But maybe we do?